docs/core/deploying/native-aot/diagnostics.md

```diff
@@ -0,0 +1,54 @@
+---
+title: AOT Diagnostics
```

Comment:
I think the below should be updated to link here:
https://github.com/dotnet/docs/blob/0c0cc1ce86cf21b4a8468bc578038bf6747baf9d/docs/core/deploying/native-aot/index.md?plain=1#L167-L169